src/misc/programs.cpp

```diff
@@ -327,7 +323,7 @@ bool Program::SetEnv(const char * entry,const char * new_string) {
 		if (!env_string[0]) break;
 		env_read += (PhysPt)(safe_strlen(env_string)+1);
 		if (!strchr(env_string,'=')) continue;		/* Remove corrupt entry? */
-		if ((strncasecmp(entry,env_string,strlen(entry))==0) && 
+		if ((strncasecmp(entry,env_string,strlen(entry))==0) &&
```

Comment:
Clang reformat. Feel free to do a full reformat pass on the whole file in a separate commit.